client/src/gamedata/en/strings.json

```diff
@@ -26,6 +26,10 @@
   "sources": "Sources",
   "submitInstance": "Submit instance",
   "getNewInstance": "Get new instance",
+  "accountNotConnectedTitle": "MetaMask account not connected",
+  "accountNotConnectedMessage": "To begin your adventure, connect your MetaMask wallet! Our game interacts directly with Ethernaut on-chain contracts, which means that your actions in the game are recorded on the blockchain. By linking your MetaMask account, you can securely engage with these smart contracts, allowing you to solve challenges, and progress in the game.",
+  "continueInReadOnly" : "Continue in read-only",
+  "connectAccount": "Connect",
```

Comment:
This string isn't currently defined for the other languages, and using another language shows a blank button instead of "Connect".

Reply:
Thank you, will fix that!